lucene/core/src/java/org/apache/lucene/index/OrdinalMap.java

```diff
@@ -359,10 +375,9 @@ public long ramBytesUsed() {
   @Override
   public Collection<Accountable> getChildResources() {
     List<Accountable> resources = new ArrayList<>();
-    resources.add(Accountables.namedAccountable("global ord deltas", globalOrdDeltas));
-    resources.add(Accountables.namedAccountable("first segments", firstSegments));
     resources.add(Accountables.namedAccountable("segment map", segmentMap));
-    // TODO: would be nice to return actual child segment deltas too, but the optimizations are confusing
+    // TODO: would be nice to return the ordinal and segment maps too, but it's not straightforward
+    //  because of optimizations.
```

Comment:
could be do something like `if (firstSegments != LongValues.ZEROES) { resources.add(Accountables.namedAccountable("first segments", firstSegments)); }`?

Reply:
I think we'd need a cast here, since `LongValues` doesn't implement `Accountable`. Alternatively, we could consider a bigger change to have `LongValues` implement `Accountable`.